src/content/docs/integrations/amazon/msk.mdx

```diff
@@ -48,10 +53,16 @@ be used by specifying respective configuration values, e.g.,
 
 ### Assuming roles
 
-Roles can also be assumed by giving the `assume_role` parameter to the `aws_iam` option.
+Roles can also be assumed by giving the `assume_role` parameter to the `aws_iam`
+option. For this, you need to create an IAM role with the necessary permission
+policy to read or write on the cluster and topic you want to use, as well as the
+ability to assume that role from your active residual role (for example the EC2
+instance role, or your local SSO developer role) via a trust relationship.
+Please refer the [Amazon MSK documentation](https://docs.aws.amazon.com/msk/latest/developerguide/security-iam.html)
+for detailed explanations.
 
 ```tql
-save_kafka "topic", aws_iam={
+to_kafka "topic", message=this.print_ndjson()+"\n", aws_iam={
```

Comment:
Why do we explicitly add `\n` here? Just to make sure it ends with newline even if the message only contains a single record?